scripts/protect_branch_git_hook.py

```diff
@@ -0,0 +1,28 @@
+import subprocess
+import sys
+
+
+def ProtectMasterBranch():
+    hookid = "protect-master-branch"
+    protected_branch = b"master"
```

Comment:
We use `main` instead of `master`. However, our primary protected branches are of the form `branch-\d+\.\d+`.